src/modules/visits/services/visits.service.ts

```diff
@@ -293,11 +293,11 @@ export class VisitsService {
 
 	public async getActiveVisitForUserAndSpace(
 		userProfileId: string,
-		spaceId: string
+		maintainerId: string
 	): Promise<Visit | null> {
 		const visitResponse = await this.dataService.execute(FIND_ACTIVE_VISIT_BY_USER_AND_SPACE, {
 			userProfileId,
-			spaceId,
+			maintainerId,
```

Comment:
Beetje verwarrend... deze werkt op maintainerId, maar in de controller staat spaceId. Ook de findById voor een space is niet de maintainerId...
als een CP in de toekomst meerdere spaces kan hebben (excuus, reading room. oops bezoekersrumte.) is dat misschien geen goed idee?

Reply:
volledig mee akkoord.
Momenteel wordt wel die andere id doorgestuurd (maintainer org id) en dat is ook de enige id die we ter beschikking hebben in elasticsearch.

We kunnen dit nog fixen, maar dat zal groter zijn dan de scope van deze PR.

Ik pas voorlopig aan zodat het overal maintainer id is